openreview/conference/webfield.py

```diff
@@ -0,0 +1,138 @@
+                merged_options[k] = default[k]
+        return merged_options
+
+    def set_landing_page(self, group, options = {}):
+
+        default_header = {
+            'title': group.id,
+            'description': ''
+        }
+
+        children_groups = self.client.get_groups(regex = group.id + '/[^/]+/?$')
+
+        links = []
+
+        for children in children_groups:
+            links.append({ 'url': '/group?id=' + children.id, 'name': children.id})
+
+        header = self.__build_options(default_header, options)
+
+        with open(os.path.join(os.path.dirname(__file__), 'templates/landingWebfield.js')) as f:
+            content = f.read()
+            content = content.replace("var GROUP_ID = '';", "var GROUP_ID = '" + group.id + "';")
+            content = content.replace("var HEADER = {};", "var HEADER = " + json.dumps(header) + ";")
+            content = content.replace("var VENUE_LINKS = [];", "var VENUE_LINKS = " + json.dumps(links) + ";")
+            group.web = content
+            return self.client.post_group(group)
+
+
+    def set_home_page(self, group, template_name, options = {}):
+
+        default_header = {
+            'title': group.id,
+            'subtitle': group.id,
+            'location': 'TBD',
+            'date': 'TBD',
+            'website': 'nourl',
+            'instructions': '',
+            'deadline': 'TBD'
+        }
+
+        header = self.__build_options(default_header, options)
+
+        with open(os.path.join(os.path.dirname(__file__), 'templates/' + template_name)) as f:
+            content = f.read()
+            content = content.replace("var CONFERENCE_ID = 'venue.org/Conference';", "var CONFERENCE_ID = '" + group.id + "';")
+            content = content.replace("var HEADER = {};", "var HEADER = " + json.dumps(header) + ";")
+            content = content.replace("var REVIEWERS_NAME = '';", "var REVIEWERS_NAME = '" + options.get('reviewers_name') + "';")
+            group.web = content
+            return self.client.post_group(group)
+
+    def set_recruit_page(self, conference_id, invitation, options = {}):
+
+        default_header = {
+            'title': conference_id,
+            'subtitle': conference_id,
+            'location': 'TBD',
+            'date': 'TBD',
+            'website': 'nourl',
+            'instructions': '',
+            'deadline': 'TBD'
+        }
+
+        header = self.__build_options(default_header, options)
+
+        with open(os.path.join(os.path.dirname(__file__), 'templates/recruitResponseWebfield.js')) as f:
+            content = f.read()
+            content = content.replace("var CONFERENCE_ID = '';", "var CONFERENCE_ID = '" + conference_id + "';")
+            content = content.replace("var HEADER = {};", "var HEADER = " + json.dumps(header) + ";")
+            invitation.web = content
+            return self.client.post_invitation(invitation)
+
+
+    def set_author_page(self, conference_id, group, options = {}):
```

Comment:
What is the purpose of the author page?

Reply:
It is like an Author Console to see all their assigned tasks for that conference